### ISSUE TITLE
Property spacing

### ISSUE THREAD

**Issue by zepfietje** [Stillat/blade-parser-typescript#5]
Property values are currently indented to make the arrows align.
I would expect the below code to be unchanged after formatting.

## Before formatting

```blade
@props([
    'some' => null,
    'property' => null,
])
```

## Currently after formatting

```blade
@props([
    'some'     => null,
    'property' => null,
])
```

## Expected after formatting

```blade
@props([
    'some' => null,
    'property' => null,
])
```

**Comment by JohnathonKoster** [comment]
Thanks for the report!

This was fixed with #9, and root-associative arrays within `@props` will now be formatted using the prettier, and will no longer force-align your properties.

**Comment by zepfietje** [comment]
Thanks, works well!
